### ISSUE TITLE
Problem with Zendesk SDK

### ISSUE THREAD

**Issue by xareelee** [hackiftekhar/IQKeyboardManager#224]
I'm using both IQKeyboardManager and [ZendeskSDK](https://github.com/zendesk/zendesk_sdk_ios). The problem is, when the RMA alert view (Rate My App) is shown and the user choose to send the feedback from the alert view, both Zendesk and IQKeyboardManager adjust the feedback view (ZDKRMAFeedbackView). I've tried to disable IQKeyboardManager when the app is launched, but it still doesn't work.

```
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  [IQKeyboardManager sharedManager].enable = NO;
  ...
}
```

I've also tried disable the view controller (**ZDKRMADialogViewController**), the result is still the same.

```
[[IQKeyboardManager sharedManager] disableInViewControllerClass:[ZDKRMADialogViewController class]];
```

Only entirely removing IQKeyboardManager from the project can make the frame of the feedback view correct.

I have no idea how to deal with it. I've also sent [the issue to ZendeskSDK (#issue 32)](https://github.com/zendesk/zendesk_sdk_ios/issues/32).

**Comment by hackiftekhar** [comment]
Thanks for the report, please let me know how can I make a demo project. I can see so many classes in Zendesk.framework and don't have proper understanding that how it should be used. If you can share a demo app then it would be great. Your help will be appreciated.

**Comment by xareelee** [comment]
Hi @hackiftekhar, thanks for your help. I'm also tracking this problem and recently I found this problem may not be due to IQKeyboardManager. If there is any update, I'll let you know.

**Comment by hackiftekhar** [comment]
May be or may be not, but keep me updated regarding this issue. Thanks :)

**Comment by hackiftekhar** [comment]
Any updates? Have you figured out the reason?

**Comment by xareelee** [comment]
@hackiftekhar Not yet. 

ZendeskSDK has published a new version. I'll update and try the SDK and again when I'm available.

**Comment by hackiftekhar** [comment]
But actually the logic depends on the client app (your app), I think it's your responsibility to disable library when showing ZendeskSDK screen and re-enable the library when ZendeskSDK dismisses. Because ZendeskSDK doesn't know who are going to integrate IQKeyboardManager and who are not. and my wordings are same for IQKeyboardManager too.

**Comment by xareelee** [comment]
Hi @hackiftekhar,

ZendeskSDK has fixed this problem in version 1.3.0.1, so I think this issue is not due to IQKeyboardManager. You could close this issue.
